### ISSUE TITLE
CMake build error

### ISSUE THREAD

**Issue by maksmakuta** [HappySeaFox/sail#178]
Hi. I'm want to build SAIL on Arch Linux, but I get this error:
```
   CMake Error at examples/c/sail-sdl-viewer/CMakeLists.txt:30 (string):
   string sub-command STRIP requires two arguments.
```
all dependencies are installed

**Comment by HappySeaFox** [comment]
As a temporary workaround, you can build with `SAIL_BUILD_EXAMPLES=OFF`

**Comment by HappySeaFox** [comment]
Fixed by e238070c
